R/internals.R

```diff
@@ -102,7 +102,7 @@ readProtoFiles2 <- function(files,
 }
 
 resetDescriptorPool <- function(){
-	.Call( "resetDescriptorPool_cpp" )
+	.Call( "resetDescriptorPool_cpp", PACKAGE = "RProtoBuf" )
```

Comment:
I was about to say we don't need this -- but then I realized that this package is still old-school not using Rcpp Attributes.  Adding this follow the code convention even though we now also register symbols so I think it is not strictly needed -- but won't harm or hurt.